lib/data_migrate/version.rb

```diff
@@ -1,3 +1,3 @@
 module DataMigrate
-  VERSION = "7.0.2".freeze
+  VERSION = "7.1.0".freeze
```

Comment:
since this is removing support for older versions of rails, this is a non-backward compatible change so it would need to be a major version bump. Also I think it should be a release candidate so that folks can test it out. So I think it should be 8.0.0.rc1

Reply:
Good point, thanks. Done